monitor-cockroachdb-with-prometheus.md

```diff
@@ -0,0 +1,154 @@
+
+3. [Add Prometheus as a datasource](http://docs.grafana.org/datasources/prometheus/), and configure the datasource as follows:
+
+   Field | Definition
+   ------|-----------
+   Name | Prometheus
+   Default | True
+   Type | Prometheus
+   Url | `http://<hostname of machine running prometheus>:9090`
+   Access | Direct
+
+4. Download the starter [Grafana dashboards](https://github.com/cockroachdb/cockroach/tree/master/monitoring/grafana-dashboards) for CockroachDB:
+   
+   ~~~ shell
+   # runtime dashboard: node status, including uptime, memory, and cpu.
+   $ wget https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/grafana-dashboards/runtime.json
+
+   # storage dashboard: storage availability.
+   $ wget https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/grafana-dashboards/storage.json
+
+   # sql dashboard: sql queries/transactions.
+   $ wget https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/grafana-dashboards/sql.json
+
+   # replicas dashboard: replica information and operations.
+   $ wget https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/grafana-dashboards/replicas.json
+   ~~~
+
+5. [Add the dashboards to Grafana](http://docs.grafana.org/reference/export_import/#importing-a-dashboard).
+
+## Step 5. Set up notifications with Alertmanager
+
+If you like, you can connect [Alertmanager](https://prometheus.io/docs/alerting/alertmanager/) to Prometheus to send notifications to email, chat clients, or other channels when specific rules are met. CockroachDB provides starter rules for your convenience.
+
+1. Download the [latest Alertmanager tarball](https://prometheus.io/download/#alertmanager) for your OS.
+
+2. Extract the binary and add it to your `PATH`. This makes it easy to start Alertmanager from any shell.
+
+3. Make sure Alertmanager installed successfully:
+
+   ~~~ shell
+   $ alertmanager -version
+   ~~~
+
+   ~~~
+   alertmanager, version 0.5.1 (branch: master, revision: 0ea1cac51e6a620ec09d053f0484b97932b5c902)
+     build user:       root@fb407787b8bf
+     build date:       20161125-08:15:17
+     go version:       go1.7.3
+   ~~~
+
+4. Download the [alerting rules](https://github.com/cockroachdb/cockroach/blob/master/monitoring/rules/alerts.rules) for CockroachDB to the `/rules` directory, where the Prometheus config expects to find it:
+   
+   ~~~ shell
+   $ wget -P rules https://raw.githubusercontent.com/cockroachdb/cockroach/master/monitoring/rules/alerts.rules
+   ~~~
+
+5. [Edit the Alertmanager config file](https://prometheus.io/docs/alerting/configuration/) that came with the binary, `simple.yml`, to specify the desired receivers for notifications.
+
+6. Start Alertmanager with the configuration file:
+
+   ~~~ shell
+   $ alertmanager -config.file=simple.yml
+   ~~~
```

Comment:
Thanks for the clarification. I added another step for adding silences via the Alertmanager UI.